src/domUpdates.js

```diff
@@ -47,6 +48,13 @@ let domUpdates = {
         destinationMenu.insertAdjacentHTML('beforeend', `<option value="${destination.id}">${destination.destination}</option>`)
       })
     }
+  },
+
+  getTotalSpent: () => {
+    domUpdates.traveler.findAllTrips(domUpdates.trips);
+    const travelTotal = document.querySelector('.travel-total');
+    const userTotal = domUpdates.traveler.calculateTravelBudget(domUpdates.destinations)
```

Comment:
domUpdates has a key of an instantiated traveler? I'm not 100% sure this is separation of DOM/data model, but, I am also `hella dumb`